src/pages/index.astro

```diff
@@ -64,16 +68,21 @@ docs.forEach((doc) => {
   });
   docIndex++;
 });
-
+}catch(e){
+  console.log(e);
+  showErrorMessage("Could not load the Articles. Please try again later.");
```

Comment:
Remove this line. We can't use it because it will be called on the server side. and there it uses `document` which isn't available on node server